Playbooks/playbook-Accessdata__Dump_memory_for_malicious_process.yml

```diff
@@ -0,0 +1,341 @@
+id: b0069573-929e-4ffc-8326-d5a9da6849f2
```

Comment:
the id and the name must be the same
Accessdata: Dump memory for malicious process